README.md

```diff
@@ -81,6 +81,10 @@ RIOT provides features including, but not limited to:
 * LoRaWAN
 * UWB
 * Bluetooth (BLE) via [NimBLE](https://github.com/apache/mynewt-nimble)
+* Rust support
+
+[Ariel OS](https://github.com/ariel-os/ariel-os) is an offspring of RIOT written
+in Rust following RIOT's targets and [vision](https://doc.riot-os.org/vision.html).
```

Comment:
I always liked littlefs' [related projects](https://github.com/littlefs-project/littlefs?tab=readme-ov-file#related-projects) section. I think it's a sign of humility to know that you are not the only game in town, but that you aspire to a friendly competition instead claiming that you are the greatest and nothing can compare.

It also helps with understanding what the project is about. "What is RIOT? Ah, it's like Zephyr or Contiki, so not something I'd run as the main OS on my laptop or in a container in the cloud."

```suggestion
### Related projects

- [Ariel OS](https://github.com/ariel-os/ariel-os) is an offspring of RIOT written
in Rust following RIOT's targets and [vision](https://doc.riot-os.org/vision.html).

- [Contiki](https://github.com/contiki-ng/contiki-ng) was an inspiration for starting RIOT, a operating system for constrained devices with a focus on networking.

- [NuttX](https://github.com/apache/nuttx) is another general purpose microcontroller OS with a focus on bringing POSIX to MCUs

- [RT-Thread](https://github.com/RT-Thread/rt-thread) is a microcontroller operating system with strong roots in China and a large community there, it has extensive support for MCUs from Chinese vendors, but also for western ones.

- [Zephyr](https://github.com/zephyrproject-rtos/zephyr) is general-purpose operating system for microcontrollers, shepparded by the Linux foundation. 

```

phrasing is of course up for suggestions, e.g. I know only very little about NuttX and RT-Thread.

My criteria for inclusion would be that it is a 'batteries included' OS with a proper networking stack.

Reply:
> I always liked littlefs' [related projects](https://github.com/littlefs-project/littlefs?tab=readme-ov-file#related-projects) section. I think it's a sign of humility to know that you are not the only game in town, but that you aspire to a friendly competition instead claiming that you are the greatest and nothing can compare.
> 
> It also helps with understanding what the project is about. "What is RIOT? Ah, it's like Zephyr or Contiki, so not something I'd run as the main OS on my laptop or in a container in the cloud."
> 
> phrasing is of course up for suggestions, e.g. I know only very little about NuttX and RT-Thread.
> 
> My criteria for inclusion would be that it is a 'batteries included' OS with a proper networking stack.

sounds like a decent suggestion. i sill would replace "offspring" by "spin-off". i also like the caption because "related" is a better fit than "alternative" (alternative really depends on the context).

Reply:
Let's go bike-shedding: I'd move Zephyr up, as I believe it to be the most obvious alternative to RIOT, one below Ariel OS.

We use [title case](https://titlecaseconverter.com/) for the other section headers, so " Related projects" --> "Related Projects".